test/framework/flux.go

```diff
@@ -588,6 +711,7 @@ func (e *ClusterE2ETest) waitForWorkerScaling(targetvalue int) error {
 	return retrier.Retry(120, time.Second*10, func() error {
 		md, err := e.KubectlClient.GetMachineDeployments(ctx,
 			executables.WithKubeconfig(e.managementKubeconfigFilePath()),
+			executables.WithNamespace("eksa-system"),
```

Comment:
todo: understand why this wasn't included previously

Reply:
It was a recent change that we realized was a bug

Reply:
Should I remove this then? I thought these objects are in the eksa-system namespace